x-pack/test/api_integration/apis/management/index_management/templates.ts

```diff
@@ -103,62 +104,69 @@ export default function ({ getService }) {
 
         // Legacy index templates
         const legacyTemplateFound = allTemplates.legacyTemplates.find(
-          (template) => template.name === legacyTemplate.name
+          (template: TemplateDeserialized) => template.name === legacyTemplate.name
         );
 
-        if (!legacyTemplateFound) {
-          throw new Error(
-            `Legacy template "${legacyTemplate.name}" not found in ${JSON.stringify(
-              allTemplates.legacyTemplates,
-              null,
-              2
-            )}`
-          );
-        }
+        expect(legacyTemplateFound).to.be.ok();
 
         const expectedLegacyKeys = [
           'name',
           'indexPatterns',
           'hasSettings',
           'hasAliases',
           'hasMappings',
-          'ilmPolicy',
```

Comment:
to re-use the helpers in serverless, we don't include ilm into the template payload by default